plugins/woocommerce/templates/checkout/form-checkout.php

```diff
@@ -50,9 +50,21 @@
 	<?php endif; ?>
 	
 	<?php do_action( 'woocommerce_checkout_before_order_review_heading' ); ?>
-	
-	<h3 id="order_review_heading"><?php esc_html_e( 'Your order', 'woocommerce' ); ?></h3>
-	
+
+	<?php
+	/**
+	 * Filter to modify the output of the order review heading.
+	 *
+	 * @since x.x.x
```

Comment:
I think just a comment describing the filter is ok here, no need for doc style comment.

Reply:
@alexflorisca Sorry to disagree, but we actually _do_ require a proper standard docblock for any new hook added, including a `@since` tag and as many `@param` tags as appropriate. 🙂 Anyway I'm requesting additional changes in a separate comment.

Reply:
Ok that's my bad. I come front a more front-end world so that's good learning for me, thank you!